onnx/test/reference_evaluator_save_test.py

```diff
@@ -0,0 +1,203 @@
+# Copyright (c) ONNX Project Contributors
+
+# SPDX-License-Identifier: Apache-2.0
+# type: ignore
+# pylint: disable=C3001,C0302,C0415,R0904,R0913,R0914,R0915,W0221,W0707
+
+import os
+import shutil
+import unittest
+
+import numpy as np
+
+from onnx import TensorProto
+from onnx.backend.test.loader import load_model_tests
+from onnx.checker import check_model
+from onnx.defs import onnx_opset_version
+from onnx.helper import (
+    make_graph,
+    make_model,
+    make_node,
+    make_opsetid,
+    make_tensor_value_info,
+)
+from onnx.reference import ReferenceEvaluator
+from onnx.reference.reference_backend import (
+    ReferenceEvaluatorBackend,
+    create_reference_backend,
+)
+
+try:
+    from onnxruntime import InferenceSession
```

Comment:
On a more general design note: It seems a bit iffy to depend on `onnxruntime` explicitly. It would be nicer if this solution were runtime agnostic.

Reply:
It does not depend on onnxruntime. There is one test with ReferenceEvaluator and the same test is run with onnxruntime if available.